### PR TITLE
Added narrower exceptions to initializeMetrics method

### PR DESCRIPTION
**Pull request by rohanshankar**
Threw NPEs and Exceptions with descriptions to the initializeMetrics method in the MetrricsUtil.java class

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java [hunk 1]
```diff
@@ -76,6 +77,8 @@ private static void initializeMetrics(boolean enabled, boolean jvmMetricsEnabled
 
       if (address != null) {
         tags.add(Tag.of("Address", address.toString()));
+      } else {
+        throw new NullArgumentException();
```
Comment (EdColeman) [inline core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java:81]: Why not IllegalArgumentException with a description - no need to add a new dependency for a standard exception.

**Review by dlmarion — CHANGES_REQUESTED** [review]
I think you may have misunderstood the task here. It was to remove the `throws Exception` from the method declaration and replace it with the actual exceptions being thrown in the method body.

**Comment by rohanshankar** [comment]
> ptions being thrown in the method body

Oh ok, sorry about that. Just to clarify, I should replace the throws Exception in the method declaration with a more specific exception that encompasses the possible exceptions thrown by anything within the method body

**Comment by ctubbsii** [comment]
Was this PR related to a specific task? If so, please mention the issue in the PR description somewhere.
Based on Dave's above comments about the misunderstanding, I'm going to close this. If I understand this correctly, this task is going to be rolled into a slightly larger change that addresses this and several other minor issues all at once. So, there's probably nothing more to do here.

**Comment by rohanshankar** [comment]
> Was this PR related to a specific task? If so, please mention the issue in the PR description somewhere. Based on Dave's above comments about the misunderstanding, I'm going to close this. If I understand this correctly, this task is going to be rolled into a slightly larger change that addresses this and several other minor issues all at once. So, there's probably nothing more to do here.

**Pull request closed without merging**